src/index.ts

```diff
@@ -314,6 +316,7 @@ export class JupiterOneClient {
     this.apiUrl = dev
       ? 'https://api.dev.jupiterone.io'
       : 'https://api.us.jupiterone.io';
+    this.apiUrl = apiBaseUrl ? apiBaseUrl : this.apiUrl;
```

Comment:
Nit, but this would have been a bit simpler.

```suggestion
    this.apiUrl = apiBaseUrl || this.apiUrl;
```